### ISSUE TITLE
Android: move Save dir to custom place

### ISSUE THREAD

**Issue by JPT77** [vcmi/vcmi#5098]
**Describe your proposal**
I would like to sync my save-games to the PC version but my phone (Fairphone 4/Android 13) does not allow custom apps to have access to the Android/data directory. 
So I would like to move the saves or the whole data dir to some custom path on int or ext SD. 
Is this hindered by android permissions?
Maybe moving to Android/media would solve the issue?

Thank you very much

**Comment by IvanSavenko** [comment]
Should be possible by connecting phone via USB. In this mode, you can access Android/data directory. 

Not most convenient method, sure, but possible.

**Comment by JPT77** [comment]
Yes, is possible via USB or via stock file manager. but both methods do not automatically sync. it's manual.

**Comment by marek-stejskal** [comment]
I am late to reply, but I would love to do the same - automatically synchronize saves from Android client with a PC. Is the possibility to move the saves to a custom folder something you can consider? As long as they are in Android/data, they are pretty much untouchable for any automation.
Many thanks!

**Comment by TheRealIndru** [comment]
If there's a way to vote for this feature, consider my vote as well. It's something we need implemented for when we play on the go vs at home. This would be a great feature!

**Comment by kambala-decapitator** [comment]
> when we play on the go vs at home

and what's the difference?

**Comment by TheRealIndru** [comment]
What do you mean? I play on my tablet on the go and PC at home. Want to move my saves around.

**Comment by GeorgeK1ng** [comment]
> > when we play on the go vs at home
> 
> and what's the difference?

Auto sync save across devices? Playing at home on Windows and have Google backup on save folder and then simply open VCMI on Android phone and continue on same save.

**Comment by Crandel** [comment]
On Android 16 it is not possible to rich Android/data at all. Please consider custom path ASAP

**Comment by Laserlicht** [comment]
> Please consider custom path ASAP

Custom path is only possible when not using play store (e.g. apk or f-droid). On play store it is only allowed for filemanagers, not for games...

**Comment by GeorgeK1ng** [comment]
Save games can be now exported / imported using Launcher.

**Comment by kambala-decapitator** [comment]
> > Please consider custom path ASAP
> 
> Custom path is only possible when not using play store (e.g. apk or f-droid). On play store it is only allowed for filemanagers, not for games...

can't you use external storage? see #3538 / https://developer.android.com/training/data-storage/app-specific#external

**Comment by Laserlicht** [comment]
> > > Please consider custom path ASAP
> > 
> > 
> > Custom path is only possible when not using play store (e.g. apk or f-droid). On play store it is only allowed for filemanagers, not for games...
> 
> can't you use external storage? see [#3538](https://github.com/vcmi/vcmi/issues/3538) / https://developer.android.com/training/data-storage/app-specific#external

Ah. Yes maybe possible. But maybe affects performance a lot. May need caching.
